src/layer/vector/Canvas.js

```diff
@@ -215,7 +215,7 @@ L.Canvas = L.Renderer.extend({
 	},
 
 	_onMouseMove: function (e) {
-		if (!this._map || this._map.dragging._draggable._moving || this._map._animatingZoom) { return; }
+		if (!this._map || (this._map.dragging._enabled && this._map.dragging._draggable._moving) || this._map._animatingZoom) { return; }
```

Comment:
I agree. Should be a simple change, but let's look if we can update similar places to use the new method.